examples/hardhat/scripts/deploy.ts

```diff
@@ -13,9 +17,20 @@ async function main() {
   // manually to make sure everything is compiled
   // await hre.run('compile');
 
+  const ethParams = calcEthereumTransactionParams({
+    gasLimit: '2100001',
+    validUntil: '360001',
+    storageLimit: '64001',
+    txFeePerGas,
+    storageByteDeposit
+  });
+
   // We get the contract to deploy
   const HelloWorld = await ethers.getContractFactory('HelloWorld');
-  const contract = await HelloWorld.deploy('Hello, Hardhat!');
+  const contract = await HelloWorld.deploy('Hello, Hardhat!', {
+    gasPrice: ethParams.txGasPrice,
```

Comment:
question: is this `gasPrice` not const anymore? If it is still const can we still put it in `hardhat.config.ts` to avoid calculating it eveytime?

Reply:
looks like it only depends on:
- storageLimit
- validUntil
- txFeePerGas
which are all const?

Reply:
This is just an example to show how to get txgasprice and txgaslimit.  Only txFeePerGas and storageByteDeposit are consts.